internal/storage/writer/writer.go

```diff
@@ -109,6 +111,37 @@ func newWriter(config *config.Compaction) (flush.Writer, error) {
 		writers = append(writers, w)
 	}
 
+	// Configure Talaria writer if present
+	if config.Talaria != nil {
+		var defaultDialTimeout time.Duration = 5
+		var defaultCircuitTimeout time.Duration = 5
+		var defaultMaxConcurrent int = hystrix.DefaultMaxConcurrent
+		var defaultErrorPercentThreshold int = hystrix.DefaultErrorPercentThreshold
+
+		// Set defaults for variables if there aren't any
+		if config.Talaria.DialTimeout == nil {
+			config.Talaria.DialTimeout = &defaultDialTimeout
+		}
+
+		if config.Talaria.CircuitTimeout == nil {
+			config.Talaria.CircuitTimeout = &defaultCircuitTimeout
+		}
+
+		if config.Talaria.MaxConcurrent == nil {
+			config.Talaria.MaxConcurrent = &defaultMaxConcurrent
+		}
+
+		if config.Talaria.ErrorPercentThreshold == nil {
+			config.Talaria.ErrorPercentThreshold = &defaultErrorPercentThreshold
+		}
+
+		w, err := talaria.New(config.Talaria.Endpoint, *config.Talaria.DialTimeout, *config.Talaria.CircuitTimeout, *config.Talaria.MaxConcurrent, *config.Talaria.ErrorPercentThreshold)
```

Comment:
Could we move the default value configurations into the talaria sink itself? Since this code is specific to talaria